harness/determined/launch/torch_distributed.py

```diff
@@ -0,0 +1,135 @@
+import argparse
+import logging
+import os
+import subprocess
+import sys
+from typing import List, Tuple
+
+import determined as det
+
+C10D_PORT = 29400
+
+
+def create_launch_cmd(
+    num_nodes: int, proc_per_node: int, node_rank: int, master_addr: str, override_args: List[str]
+) -> List[str]:
+    cmd = [
+        "python3",
+        "-m",
+        "torch.distributed.run",
+        "--nnodes",
+        str(num_nodes),
+        "--nproc_per_node",
+        str(proc_per_node),
+        "--node_rank",
+        str(node_rank),
+        "--max_restarts",
+        "0",
+        "--master_addr",
+        master_addr,
+        "--master_port",
+        str(C10D_PORT),
+        "--module",
```

Comment:
i think we should exclude the --module argument from the launch command and instead support it as an override arg.  what do you think?

Reply:
done

Reply:
done